### ISSUE TITLE
[Bug]: Field doesn't announce validation message when using Narrator and navigating via keyboard

### ISSUE THREAD

**Issue by WillHavelin** [microsoft/fluentui#32972]
### Component

Field

### Package version

9.54.17

### React version

18.3.1

### Environment

```shell
This is from the StackBlitz for the reproduction
System:
    OS: Linux 5.0 undefined
    CPU: (8) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
    Memory: 0 Bytes / 0 Bytes
    Shell: 1.0 - /bin/jsh
  npmPackages:
    @fluentui/react-components: ^9.0.0 => 9.54.17 
    @fluentui/react-icons: latest => 2.0.260 
    @types/react: ^17 => 17.0.83 
    @types/react-dom: ^17 => 17.0.25 
    react: ^18 => 18.3.1 
    react-dom: ^18 => 18.3.1
```

### Current Behavior

When using Narrator and tabbing (or shift tabbing) to Field components with validation messages the validation messages are not announced (with error validationState it will say invalid, but not the actual error message)

### Expected Behavior

I would expect the screen reader to announce the validation message, especially for warnings or errors. Otherwise, a user who is using a screen reader will not be able to easily understand what is invalid about the current input in the field. 

### Reproduction

https://stackblitz.com/edit/9fjocv?file=src%2Fexample.tsx

### Steps to reproduce

1. Turn on Narrator (or another screen reader software)
2. Tab and shift tab through the fields

### Are you reporting an Accessibility issue?

yes

### Suggested severity

High - No workaround

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by TristanWatanabe** [comment]
@WillHavelin I'm unable to reproduce the error in the latest version of `@fluentui/react-components` (version `9.55.0`) in either Edge + Narrator or Edge + NVDA browser/screenreader combinations. Is this still an issue?

https://github.com/user-attachments/assets/4ea110dd-b867-44c9-abdb-5792067687ac

**Comment by WillHavelin** [comment]
@TristanWatanabe Hi, thanks for looking into this. I tried with Edge and Narrator and it seems to be working correctly, but I am still seeing the error with Chrome and Narrator

**Comment by dmytrokirpa** [comment]
It appears that the issue where the element with an id from `aria-describedby` is not announced occurs only with the Narrator + Chrome combination. Other combinations such as Narrator + Edge, NVDA + Edge/Chrome, and Edge/Chrome/Safari + VoiceOver work as expected and the field error/description announced properly.


I'm unsure if there is anything we can do from the FluentUI perspective to fix the issue.

Also a similar issue was reported [here](https://techcommunity.microsoft.com/discussions/windowsinsiderprogram/narrator-is-not-reading-aria-labelledby-and-aria-describedby-in-chrome-browser/2589607)


@smhigley What would you recommend to do in such case?
